tests/codes/eurocode/nen_en_1992_1_1_c2_2011/test_chapter_7_serviceability_limit_state.py

```diff
@@ -0,0 +1,60 @@
+"""Testing module for chapter 7 of NEN-EN 1992-1-1+C2:2011."""
```

Comment:
Is there a convention of placing all chapter 7 formula tests in one file? I personally prefer to test eacah object in a single file for a nice overview, also if you run Pytest, you can see which files fail in one eye sight